roll: 0
pitch: 0
yaw: 270

x_pos: 1
y_pos: 1
z_pos: 1